docs/manual/common/guide/devmode/DevEnvironment.md

```diff
@@ -90,9 +90,28 @@ This all happens automatically without special code or additional configuration.
 You can verify that your services are running by viewing `http://localhost:9008/services` in a web browser (or with a command line tool such as `curl`). The Service Locator, running on port `9008`, will return JSON such as:
 
 ```
-[{"name":"hello-stream","url":"http://127.0.0.1:26230"},
- {"name":"cas_native","url":"tcp://127.0.0.1:4000/cas_native"},
- {"name":"hello","url":"http://127.0.0.1:24266"}]
+[
+  {
+    "name":"cas_native",
+    "url":"tcp://127.0.0.1:4000/cas_native",
+    "portName":null
```

Comment:
So, `null` means that it will default to something? If so, to what? And what if I don't want the default?

I remember some discussion about this, but I don't know if the result of such conversation was translated into our docs.

Reply:
`null` is used as a placeholder for "if the client doesn't ask for a specific `portName` return this row of the registry". It's the equivalent to a non-SRV `Lookup(...)` that returns `A/AAAA` on `DNS`.

Reply:
I don't think it's worth documenting. This is anecdotical docs.

Reply:
I'm wondering why didn't we make it an `Option` instead. Anyway, I don't think it's worth going down that path. This is expected to become obsolete in the future.

Reply:
> I'm wondering why didn't we make it an `Option` instead.

Speculating: at least for Play, all the settings must be defined, and if they don't have a value, then configured to null (and later read as an `Option[T]`. Although this is not reading from `conf` files, it feels more consistent this way.